src/AppBundle/Model/Event.php

```diff
@@ -49,6 +49,8 @@ class Event
         'byte-difference' => null,
         'pages-created' => null,
         'pages-improved' => null,
+        'pages-created-pageviews' => null,
```

Comment:
This doesn't need an offset too? I think we only use them in messaging, which I know doesn't exist yet for this metric.

Reply:
Nope, the `pages-improved-pageviews` is an average per day over (up to) the last 30 days, whereas the `pages-created-pageviews` is the cumulative total over all time since page creation till now (so there's no need for an offset).